README.md

```diff
@@ -24,7 +24,7 @@ import Container from '@hixme-ui/container'
 import Title from '@hixme-ui/title'
 
 const AppContainer = () => (
-  <ThemeProvider>
+  <ThemeProvider withGlobal>
```

Comment:
Cool stuff, I dig this.

Reply:
thought this would be a good way to opt in and update explicitly